tap_google_search_console/client.py

```diff
@@ -183,7 +186,8 @@ def __init__(self,
                  client_secret,
                  refresh_token,
                  site_urls,
-                 user_agent=None):
+                 user_agent=None,
+                 timeout_from_config=None):
```

Comment:
@harshpatel4crest Can we not keep the default value of timeout_from_config as `REQUEST_TIMEOUT`.
Please follow a standard code practice across all the taps.
Reference: https://github.com/singer-io/tap-recharge/pull/19/files#diff-6c1c4269f0b2a173853e6ebc52d1597c0b1a349842952fe9036458e26028c4f7R106